scripts/release.js

```diff
@@ -136,7 +136,7 @@ prompt([
         fs.writeFileSync(changelogPath, newChangelog, 'utf8');
 
         // Rebuild the docs
-        if (!newVersion.indexOf('alpha')) {
```

Comment:
This was checking if the result of `indexOf` is falsey, but `indexOf` returns `-1` or a positive index, both of which resolve to false:

```
> !(-1)
false
> !(1)
false
```